pcdsdevices/lasers/btps.py

```diff
@@ -165,11 +165,11 @@ class GlobalConfig(BaseInterface, Device):
 
     min_centroid_change = Cpt(
```

Comment:
Picking nits, but this name doesn't seem consistent now that we're looking at the image sum.

Reply:
Less a nitpick and more a mistake 😁 

```suggestion
    min_sum_change = Cpt(
```